lib/leaf.js

```diff
@@ -34,20 +36,28 @@ var Leaf = React.createClass({
         };
 
         var onLabelClick = this._onClick.bind(this, d);
-
+        
         return D.div({ className: this.getClassName(), id: 'leaf-' + this._rootPath() },
             D.input({ className: 'json-inspector__radio', type: 'radio', name: p.id, id: id, tabIndex: -1 }),
             D.label({ className: 'json-inspector__line', htmlFor: id, onClick: onLabelClick },
                 D.div({ className: 'json-inspector__flatpath' },
                     d.path),
-                D.span({ className: 'json-inspector__key' },
-                    this.format(d.key),
-                    ':',
-                    this.renderInteractiveLabel(d.key, true)),
+                this.renderLabel(d.key),
                 this.renderTitle(),
                 this.renderShowOriginalButton()),
             this.renderChildren());
     },
+    renderLabel: function(key){
+        var leafLabel = D.span({ className: 'json-inspector__key' },
+                    this.format(key),
+                    ':',
+                    this.renderInteractiveLabel(key, true));
+        if(this.props.root && !this.props.showRootLabel){
+            leafLabel = D.span({ className: 'json-inspector__key' },
+                    this.renderInteractiveLabel(key, true));
+        }
+        return leafLabel;
+    },
```

Comment:
Sounds great. Thanks for looking at my changes and thanks for the great library :-)